great_expectations/core/config_provider.py

```diff
@@ -6,16 +6,16 @@
 from collections import OrderedDict
 from typing import Any, Dict, Optional, Type, cast
 
-from great_expectations.core.config_substitutor import ConfigurationSubstitutor
+from great_expectations.core.config_substitutor import _ConfigurationSubstitutor
 from great_expectations.core.yaml_handler import YAMLHandler
 from great_expectations.data_context.types.base import GXCloudConfig
 
 yaml = YAMLHandler()
 
 
-class AbstractConfigurationProvider(ABC):
+class _AbstractConfigurationProvider(ABC):
```

Comment:
I will admit now that I'm seeing it.... Having private ABC's seems a bit odd.
But I don't have the words to express why 😆

Reply:
I can undo this if we don't like it! Don't feel strongly either way 😄